raiden/raiden_service.py

```diff
@@ -418,7 +413,6 @@ def start(self) -> None:
         self._initialize_transactions_queues(chain_state)
         self._initialize_messages_queues(chain_state)
         self._initialize_whitelists(chain_state)
-        self._initialize_channel_fees()
```

Comment:
I see, you're removing setting channel fees at startup / restart. How do you persist the channel fee settings? How do you make sure, that channel fee settings are (or have been!) communicated to the selected PFS?